extensions/timers/02_performance.js

```diff
@@ -180,16 +246,19 @@
       startTime,
       duration,
       detail = null,
-      key,
+      key = undefined,
```

Comment:
Add defaults to all constructor arguments so they don't count towards `PerformanceMeasure.length`.